be/test/formats/parquet/parquet_cli_reader_test.cpp

```diff
@@ -137,6 +137,7 @@ TEST_F(ParquetCLIReaderTest, ReadArrowFuzzingParquetFiles) {
         if (ignore_dcheck_paths.find(path) != ignore_dcheck_paths.end()) {
             continue;
         }
+        std::cout << "Testing file: " << path << std::endl;
```

Comment:
turn to logging instead of std::cout

Reply:
@kevincai Since this is a just ut, can I fix that in following PR?